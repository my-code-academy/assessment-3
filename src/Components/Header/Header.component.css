
.header {
  width: 100%; 
  height: 20px; 
  border-bottom: 5px solid white; 
  text-align: center;
  background-color: #494748;
}

.headerText {
  font-size: 3rem; 
  background-color: #494748; 
  padding: 0 10px;
  color: white;
}