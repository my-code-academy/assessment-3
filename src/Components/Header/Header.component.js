import React, { Component } from 'react';
import './Header.component.css';

class HeaderComponent extends Component {
  render() {
    return (
      <div className='header'>
        <span className='headerText'>
          The book Shelf 
        </span>
      </div>
    )
  }
}
 export default HeaderComponent;
