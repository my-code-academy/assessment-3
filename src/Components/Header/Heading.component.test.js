import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Header from './Header.component';

describe('Header', () => {
  it ('should match the snapshot', () => {
    const tree = renderer.create(<Header />);
    expect(tree).toMatchSnapshot();
  });
})