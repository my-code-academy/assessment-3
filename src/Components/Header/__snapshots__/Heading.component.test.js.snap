// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  className="header"
>
  <span
    className="headerText"
  >
    The book Shelf
  </span>
</div>
`;
